recipes/nncf/meta.yaml

```
@@ -0,0 +1,60 @@
+{% set name = "nncf" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8bdfabaf1b163da6e80dc2d7906c2c8342567cff28c40e97c2cb65929a750736
+
+build:
```

```suggestion
build:
  skip: True  # [win]
```

I'd skip windows if the build there is not going to work.